### PR TITLE
Add cosmoshub-4 to registry.yaml

### PR DESCRIPTION
**Pull request by orkunkl**

### CODE REVIEW

File: demo/registry.yaml [hunk 1]
```diff
@@ -50,3 +43,11 @@ chains:
     faucet: http://49.12.73.189:8002
     rpc:
       - http://188.34.162.78:26657
+  cosmoshub-4:
+    chain_id: cosmoshub-4
+    prefix: cosmos
+    gas_price: 0.01uatom
```
Comment (ethanfrey) [inline demo/registry.yaml:49]: Are you sure this gas price works?
I have used `0.025uatom` as a default for some time. Happy if it is cheaper now.

**Review by ethanfrey — APPROVED** [review]
Looks good.
Please verify the gas price works.

File: demo/registry.yaml (hunk 1 above)
Comment (orkunkl) [reply demo/registry.yaml]: Yeah I made it `0.026uatom` after some tx timeouts. I was not sure what was the default uatom gas price among cosmos hub validators.

File: demo/registry.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ chains:
   cosmoshub-4:
     chain_id: cosmoshub-4
     prefix: cosmos
-    gas_price: 0.01uatom
+    gas_price: 0.026uatom
```
Comment (ethanfrey) [inline demo/registry.yaml:49]: :+1:

**Review by ethanfrey — APPROVED** [review]
Better. Maybe add a second rpc endpoint for docs (even if we only use one)

**Comment by orkunkl** [comment]
I reverted musselnet registry comments

**Pull request merged**